accepted/2021/experimental-package-shading.md

```diff
@@ -0,0 +1,73 @@
+# Experiment: NuGet Package Shading
+
+(intro and scenarios to be added)
+
+## Usage
+
+### Package authors
+
+A *package author* may enable shading on any package reference as long as its assets are configured such that it is effectively private, and as long as any transitive references to that package are via shaded package references. When they restore their project, its shaded package references will be substituted for renamed copies of those packages and their assets. When the project is packed, these renamed assets are bundled in the resulting package.
+
```

Comment:
> A *package author* may enable shading on any package reference 

"How?" A walkthrough of experience is needed?

Reply:
May be link to the "Controlling Shading" section from here is that's where the actual experience is explained.

Reply:
Added MSBuild snippet here

Reply:
This now has an MSBuild example.